OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -90,6 +90,7 @@ public class ProductionQueue : IResolveOrder, ITick, ITechTreeElement, INotifyOw
 		[Sync] public bool Enabled { get; private set; }
 
 		public string Faction { get; private set; }
+		public bool autoqueue { get; private set; }
```

Comment:
This will trigger StyleCop for sure. Public members should be capitalized (as in `AutoQueue`) and in this case probably even synced as well.